components/AvatarWithLink.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Account } from '../lib/graphql/types/v2/graphql';
+import { getAvatarBorderRadius } from '../lib/image-utils';
+
+import Avatar from './Avatar';
+import LinkCollective from './LinkCollective';
+
+type AvatarWithLinkProps = {
+  account: Account;
+  secondaryAccount: Account | null;
+  /** The size in pixels */
+  radius: number;
+};
+
+const DualAvatarContainer = styled.div`
+  position: relative;
+`;
+
+type SecondaryAvatarContainerProps = {
+  borderRadius: string | number;
+};
+
+const SecondaryAvatarContainer = styled.div<SecondaryAvatarContainerProps>`
+  position: absolute;
+  background: white;
+  border-radius: 6px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  bottom: -8px;
+  right: -8px;
+  width: 50%;
+  height: 50%;
+  border-radius: ${({ borderRadius }) => borderRadius};
```

Comment:
Since we're adding a white background (that looks like a border around the avatar), we need to ensure it has the same border-radius as the child avatar